### ISSUE TITLE
Exported statics aren't declared in the generated header

### ISSUE THREAD

**Issue by WrinklyNinja** [eqrion/cbindgen#47]
I've got some Rust code looking like this:

```rust
#[no_mangle]
pub static LIBLO_OK: c_uint = 0;

#[no_mangle]
pub static LIBLO_ERROR_INVALID_ARGS: c_uint = 12;
```

These are used as return codes for some of my API functions, but when cbindgen runs it doesn't include their declarations in the header. I'd expect it to include them like this:

```c
extern unsigned int LIBLO_OK;
extern unsigned int LIBLO_ERROR_INVALID_ARGS;
```

If I add the above block to cbindgen's `trailer` config string, I can then use the constants in the C code that calls my library, but isn't this the sort of thing that cbindgen should be doing automatically?

**Comment by eqrion** [comment]
Yeah cbindgen should definitely be able to handle this.

Finding static declarations won't be too challenging, the harder part is knowing which ones to output. Right now we only output declarations that are using by the FFI, but that won't work here.

I suppose we could just output all static items with `#[no_mangle]` as they are all probably meant to be exported.

**Comment by WrinklyNinja** [comment]
> I suppose we could just output all static items with #[no_mangle] as they are all probably meant to be exported.

That's the behaviour I'd expect, I don't know of any reason for not mangling names other than for FFI export.

**Comment by RazrFalcon** [comment]
@eqrion what about `const`?

**Comment by Michael-F-Bryan** [comment]
> @eqrion what about const?

I was about to open an issue for this, but looks like it's already been mentioned. I reckon `const` statements should be converted to a `#define`. This'll work fine for integers and strings, but I haven't put much thought into more complex types...

**Comment by RazrFalcon** [comment]
At least numbers support will be fine for now.

**Comment by eqrion** [comment]
Fixed in 0.1.27. Please reopen if you're still having an issue with this.

**Comment by WrinklyNinja** [comment]
Thanks 👍
